beacon-chain/rpc/validator/status.go

```diff
@@ -159,11 +160,22 @@ func (vs *Server) validatorStatus(
 			log.Warn("Not connected to ETH1. Cannot determine validator ETH1 deposit block number")
 			return resp, nonExistentIndex
 		}
-		_, eth1BlockNumBigInt := vs.DepositFetcher.DepositByPubkey(ctx, pubKey)
+		deposit, eth1BlockNumBigInt := vs.DepositFetcher.DepositByPubkey(ctx, pubKey)
 		if eth1BlockNumBigInt == nil { // No deposit found in ETH1.
 			return resp, nonExistentIndex
 		}
-
+		domain, err := helpers.ComputeDomain(
+			params.BeaconConfig().DomainDeposit,
+			nil,
```

Comment:
In general, it's a good practice to add a comment after `nil,` so one knows what this nil refers to

Reply:
done